backend/database/repositories/frontends.go

```diff
@@ -1,15 +1,32 @@
 package repositories
 
+import (
+	"github.com/google/uuid"
+)
+
 type frontendsRepository struct {
 	embeddedContext
 }
 
 const InvalidFrontend = -1
 
+func NewFrontendRepo(frontEndID uuid.UUID, logicalName string, URL string, embeddedContext embeddedContext) (filesystemRepository, error) {
+	err := embeddedContext.ctx.Query("SELECT new_frontend($1, $2, $3)", []interface{}{frontEndID, logicalName, URL}, &frontEndID)
+	if err != nil {
+		return filesystemRepository{}, err
```

Comment:
I feel like moving on there should be better handling of errors, perhaps we should wrap this error up in a secondary error to decorate it with a bit more context